tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -1618,6 +1619,23 @@ LogicalResult ConvertTFLTransposeConvOp::matchAndRewrite(
   bool output_is_qtype =
       output_type.getElementType().isa<mlir::quant::QuantizedType>();
 
+  const bool has_bias = !(!tfl_conv_op.getBias() ||
```

Comment:
Isn't this

tfl_conv_op.getBias() && !isa<NoneType>(tfl_conv_op.getBias().getType())

? (it reads slightly easier to me than the double negative)

Reply:
Yes agreed, this is done